exercises/etl/src/example.c

```diff
@@ -0,0 +1,47 @@
+#include "etl.h"
+#include <stdlib.h>
+#include <string.h>
+#include <ctype.h>
+#include <stdio.h>
+
+static int compare(const void *a, const void *b)
+{
+   new_map *p = (new_map *) a;
+   new_map *q = (new_map *) b;
+   return (p->key - q->key);
+}
+
+int convert(legacy_map * input, int input_len, new_map ** output)
```

Comment:
Is it possible to convert the parameter types so that the interface is simpler: `int convert legacy_map input[], int input_len, new_map * output[]);`.